README.md

```diff
@@ -10,49 +10,246 @@ It has only one function: generate the [Fibonacci sequence](https://en.wikipedia
+
+\> Alternatively, you could configure the workflow to [trigger when a tag is pushed](https://stackoverflow.com/a/61892639) to your repository.
+
+\> The script above disables Gradle [configuration cache](https://docs.gradle.org/current/userguide/configuration_cache.html) for the publication task by adding `--no-configuration-cache` to the Gradle command, as the publication plugin does not support it (see this [open issue](https://github.com/gradle/gradle/issues/22779)).
+
+\> Reminder: When using `publishToMavenCentral`, you’ll still need to check and release your deployment manually on the website, as described in the previous section. You may use `publishAndReleaseToMavenCentral` instead for a fully automated release.
+
+This action will need your signing details and your Maven Central credentials. These will be configured as GitHub Actions secrets in the next section. The configuration of the workflow above takes these secrets and places them into environment variables, which will make them available to the Gradle build automatically.
+
+### Add secrets to GitHub
+
+To use the keys and credentials required for publication in your GitHub Action workflow while keeping them private, you need to place those values into secrets. From your GitHub repository, go to `Settings` \> `(Security) Secrets and variables > Actions`.
+
+Click on the `New repository secret` button, and add the following secrets:
+
+- `MAVEN_CENTRAL_PASSWORD` and `MAVEN_CENTRAL_PASSWORD` are the values generated by the Central Portal website in the [Generate User Token](#generate-the-user-token) section.
+- `SIGNING_KEY_ID` is **the last 8 characters** of your signing key’s identifier.
+- `SIGNING_PASSWORD` is the passphrase you’ve provided when generating your signing key.
+- `GPG_KEY_CONTENTS` should contain the contents of your GPG private key file, which you’ve created earlier in the [Export your private key](#export-your-private-key) section.
+
+#
+
+![](/images/image5.png)
+
+Note again that the names used for these secrets must match those used by the workflow that accesses their values.
+
+#### Create a release on GitHub
+
+With the workflow and secrets set up, you’re now ready to [create a release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) that will trigger the publication of your library.
+
+Go to your GitHub repository’s main page, and click on Releases in the menu in the right sidebar.
+
+![](/images/image3.png)
+
+Click *Draft a new release*.
+
+![](/images/image7.png)
+
+Each release creates a new tag. Set the name for the tag to be created, and set a name for the release (these may be identical). Note that setting a version here does not change the version of your coordinates configured in your `build.gradle.kts` file, so you should update that version before creating a new release.
```

Comment:
I haven't used the vanniktech plugin, but usually the projects group/name/version are automatically picked up, so they shouldn't need to be specified twice.

I think that recommending that they are specified twice could be a slight touch overkill, however, I recognise that sometimes being explicit is nice, even if it's unnecessary.

So, I would like to investigate to see just setting `group = "..."` and `version = "..."` is enough, and if so, then remove the duplicate config. But if you think being explicit is good, then we can leave it as is.